src/Tokenizer/Token.php

```diff
@@ -20,7 +20,10 @@
  * Representation of single token.
  * As a token prototype you should understand a single element generated by token_get_all.
  *
+ * @immutable
+ *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Michael Vorisek <https://github.com/mvorisek>
```

Comment:
only if picky - quite an addition for adding one annotation in this PR :D

Reply:
Small class, small changes :D